bot/src/main/resources/translations/melijn/strings.properties

```diff
@@ -212,3 +212,38 @@ botCannotInteractWithRole=I cannot interact with {0}
 role.give.gaveRole=Gave the {0} role to **{1}**
 role.take.tookRole=Took the {0} role from **{1}**
 
+anilist.link.nouser=No such user found
```

Comment:
some of these should have their casing fixed (e.g. nouser) I'd read this as nous-er xd
and the single letter abbreviation can be more verbose: eDate might be confused with online dating instead of the end date